src/main/java/eu/usrv/enhancedlootbags/core/items/ItemLootBag.java

```diff
@@ -89,7 +88,8 @@ public String getItemStackDisplayName(ItemStack pStack) {
         if (pStack.getItemDamage() == 0) tInnerName = StatHelper.get("string.default");
         else {
             LootGroup tGrp = _mLGHandler.getGroupByIDClient(pStack.getItemDamage());
-            tInnerName = tGrp == null ? "Error" : tGrp.getGroupName();
+            String tLocale = StatHelper.get("group." + tGrp.getGroupName().toLowerCase().replaceAll("\\s+", "_"));
+            tInnerName = tGrp == null ? "Error" : (tLocale == null ? tGrp.getGroupName() : tLocale);
```

Comment:
Here you introduce a `NullPointerException` since you dereference `tGrp` before checking if it's null. You should instead replace `tGrp.getGroupName()` in the original code by a call to a new method which takes in the group name and localizes it if possible. That return value shouldn't possibly be null so there would be no need to check for it being null as is being done here with `tLocale == null ?`.